packages/store/package.json

```diff
@@ -38,6 +38,8 @@
   },
   "devDependencies": {
     "@agoric/swingset-vat": "^0.28.0",
+    "@agoric/ertp": "^0.14.2",
```

Comment:
I think this module should not take a dependency on zoe or ertp. Instead, copy the shapes, since they just need to be representative rather than accurate.

Reply:
Done.